api-doc.md

```diff
@@ -388,3 +388,32 @@
     - Headers : none
     - Message : Invitation mail has been sent successfully
 ```
+
+### 8. Send Acceptence Mail (Notification)
+
+```
+- Method : POST
+- Endpoint : /api/acceptence
+- Description : API for sending Accepted mail to the Company HR manager
+- Request:
+    - Headers : none
+    - Body :
+         {
+            "receiver": "hr@canopas.com",
+            "companyname" : "canopas"
```

Comment:
No need for company name, as the company's HR get an email for their company only